src/assets/json/communities.json

```diff
@@ -258,7 +258,42 @@
     "organizers": [
       {
         "name": "Michael Hladky",
-        "twitter": "https://twitter.com/michael_hladky"
+        "twitter": "https://twitter.com/michael_hladky",
+        "webs": ["https://www.linkedin.com/in/michael-hladky-519340148/"]
+      },
+      {
+        "name": "Miroslav Jonas",
+        "twitter": "https://twitter.com/meeroslav/",
+        "webs": ["https://missing-manual.com/"]
+      }
+    ]
+  },
+  "Angular Austria": {
+    "id": 87,
+    "name": "Angular Austria",
+    "position": {
```

Comment:
2. Change the position because Aungular-Austria and Angular-Viena have the same latitude/longitude. The communities must to have different latitude/longitude